naucse/notebook_util.py

```diff
@@ -0,0 +1,43 @@
+from nbconvert import HTMLExporter
+from nbconvert.filters.highlight import Highlight2HTML, _pygments_highlight
+import nbformat
+import traitlets
+import pygments
+
+from . markdown_util import convert_markdown
+
+
+class Highlight2HTMLCodehilite(Highlight2HTML):
+    '''We need to do all this just to add the codehilite cssclass'''
```

Comment:
Using `_pygments_highlight` sounds like asking for trouble.

Can we switch to using the same highlighter code as the rest of naucse, or does `_pygments_highlight` do something special?

Alternately, we don't need to use `codehilite` for the class. It was a default from the old Markdown parser. We could just switch to `highlight` if that's what Notebook uses by default.
I can elaborate, if that sounds like a good idea.